aci/data_source_aci_pimifpol.go

```diff
@@ -0,0 +1,83 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAciPIMInterfacePolicy() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   dataSourceAciPIMInterfacePolicyRead,
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"tenant_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"auth_key": {
```

Comment:
Is the key returned from the API? Should it be marked sensitive?

Reply:
I removed it from the resource because it was not returned from the API